.circleci/config.yml

```diff
@@ -4,23 +4,57 @@
 version: 2
 jobs:
   build:
-    docker:
-      - image: circleci/golang:1.11
-    working_directory: /go/src/github.com/litmuschaos/chaos-operator
+    machine:
+        image: circleci/classic:201808-01
+        docker_layer_caching: true
+    environment:
+      K8S_VERSION: v1.12.0
+      KUBECONFIG: /home/circleci/.kube/config
+      MINIKUBE_VERSION: v1.3.1
+      MINIKUBE_WANTUPDATENOTIFICATION: false
+      MINIKUBE_WANTREPORTERRORPROMPT: false
+      MINIKUBE_HOME: /home/circleci
+      CHANGE_MINIKUBE_NONE_USER: true 
+
+    working_directory: ~/go/src/github.com/litmuschaos/chaos-operator
+    
     steps:
       - checkout
+      - run:
+         name: Setup kubectl
+         command: | 
+           curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+      - run:
+          name: Setup minikube
+          command: |
+            curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64  && chmod +x minikube
+            sudo install minikube /usr/local/bin
+      - run:
+          name: Start minikube
+          command: |
+            sudo -E minikube start --vm-driver=none --cpus 2 --memory 4096 --kubernetes-version=${K8S_VERSION}
+            cat /home/circleci/.kube/config
+  
       - run: 
           name: Setup ENV
           command: |
-            echo 'export GOPATH=/go' >> $BASH_ENV
+            echo 'export GOPATH=/home/circleci/go' >> $BASH_ENV
+            echo $?
             echo 'export PATH="$GOPATH/bin:$PATH"' >> $BASH_ENV
             echo 'export REPONAME=litmuschaos' >> $BASH_ENV
             echo 'export IMGNAME=chaos-operator' >> $BASH_ENV
-            echo 'export IMGTAG=ci' >> $BASH_ENV
+            go env
+            ls && pwd
+            echo 'export IMGTAG=latest' >> $BASH_ENV
+            
+            
+  
       - run: make deps
       - run: make gotasks
-      - setup_remote_docker:
-          docker_layer_caching: true 
       - run: |
-          docker build . -f build/Dockerfile -t ${REPONAME}/${IMGNAME}:${IMGTAG}
+          docker build . -f build/Dockerfile -t litmuschaos/chaos-operator:commit-hash
```

Comment:
Instead of directly using the repo name and image can you use via an environment variable, also the tage is not commit-hash it's the commit id of the PR upto 8 charactor